marxs/source/labSource.py

```diff
@@ -3,15 +3,28 @@
 
 from .source import Source
 from ..optics.base import FlatOpticalElement
+from ..optics.polarization import polarization_vectors
 
 
-class FarLabConstantPointSource(Source, FlatOpticalElement):
+class PolarizedSource(Source):
+    def __init__(self, **kwargs):
```

Comment:
I would not all this a "polarized source". It makes light with random polarization vectors, which is exactly what we usually call an **Un**polarized source. A better description would be "Source with polarization vectors filled in instead of empty". I really think that this is so general, that the `random_polarization` should just be added to the general `Source`. Then, there is no need for this `PolarizedSource` at all.

If people really have a source that intrinsically makes polarized light (e.g. an astrophysical maser), then can override the `random polarization`.